### PR TITLE
[Runtime] Android argsort support

### PR DESCRIPTION
**Pull request by jwfromm**
Simple change on top of the android contrib fix by @tqchen to enable argsort in the android runtime. See [this discussion](https://discuss.tvm.ai/t/using-contrib-libraries-with-android/3151/7) for more details on this fix.

### CODE REVIEW

File: apps/android_rpc/app/src/main/jni/Application.mk [hunk 1]
```diff
@@ -23,3 +23,7 @@ ifeq ($(USE_VULKAN), 1)
     APP_CPPFLAGS += -DTVM_VULKAN_RUNTIME=1
     APP_LDFLAGS += -lvulkan
 endif
+
+ifeq ($(USE_SORT), 1)
+	APP_CPPFLAGS += -DUSE_SORT=1
```
Comment (yzhliu) [inline apps/android_rpc/app/src/main/jni/Application.mk]: let's use 4-space instead

Comment (jwfromm) [reply apps/android_rpc/app/src/main/jni/Application.mk]: whoops, replaced with spaces.

Comment (jwfromm) [reply apps/android_rpc/app/src/main/jni/Application.mk]: Also split off this commit from Tianqi's changes.

**Comment by yzhliu** [comment]
Thanks @jwfromm !

**Pull request merged**